about.html

```diff
@@ -0,0 +1,141 @@
+        <li><a class="close" href="#">Service</a></li>
+        <li><a class="close" href="#">Blogs</a></li>
+        <li><a class="close" href="#">Support</a></li>
+        <li><a class="close" href="#">News</a></li>
+        <li class="bordered-link"><a class="close" href="#">CONTACT US</a></li>
+      </ul>
+    </div>
+  </header>
+
+  <main class="main-section-container">
+    <div class="transparent">
+      <div class="main-section">
+        <h2 class="title-intro">
+           <q>Welcome!! TechnoCamp United Kingdom 2022</q>
+        </h2>
+        <h1 class="title-info">Creative Minds Global Summit UK</h1>
+      <div class="description">
+        <p>People meet here has the great minds and aspirantion in leading the web development from front into a next Level.
+          Our goal is to assist the client in maintaing websites more secured way, 
+          less maintainence cost and workforce in delievering greater performance.</p>
+      </div>
+      <div class="office">
+        <p>Please contact us on Email to Know any additional Information</p>
+        <h2>info@technocamp2022.co.uk</h2>
+      </div>
+     </div>
+    </div>
+  </main>
+
+  <section class="camp-work">
+    <h2 class="camp-title">TechnoCamp2022</h2>
+    <hr class="camp-title-separators">
+    <p class="camp-description">
+      We are a dedicated team stands for providing customers with dependable and most valuable services, 
+      and we are delighted to have safeguarded thousands of websites over the last two years.
+       We have the highest customer satisfaction rating in the large scale web application. 
+      You can count on us to be there for you when you need us the most. 
+      Our price choices are affordable and competitive.
+    </p>
+    <div class="camp-logo">
+      <img src="./Images/tech-camp-logo.png" alt="logo">
+    </div>
+  </section>
+
+  <section class="camp-work">
+    <h2 class="camp-title"> See Previous TechnoCamp Meetup</h2>
+    <hr class="camp-title-separators">
+    <div class="meeting">
+      <div class="meetup-1">
+        <ul class="meeting-outline">
+          <li>2021</li>
+          <li>TechnoCamp2021</li>
+          <li>Website Modernisation</li>
+        </ul>
+      </div>
+      <div class="meetup-2">
+        <ul class="meeting-outline">
+          <li>2020</li>
+          <li>TechnoCamp2020</li>
+          <li>Website Security</li>
+        </ul>
+      </div>
+  </div>
+  </section>
+  
+  <footer>
+    <div class="footer-section">
+      <div class="confer-logo">
+        <a href="./index.html" class="logo-footer">
+          <img class="footer-logo" src="./Images/tech-camp-logo.png" alt="Techno_Logo">
+         </a>
+      </div>      
+      <div class="confer-address">
+        <p>Oxford square Building,
+           Conference Hall Number: 304, 
+           Headington,Oxford,Oxfordshire, United Kingdom - OXG 1FG</p>
+      </div>
+    </div>   
+  </footer>
```

Comment:
- [ ] On desktop size the footer has a white background whereas on the mobile size the footer has a white background. Please make this change on both pages.